apps/storybook/src/NotificationMarker.stories.tsx

```diff
@@ -0,0 +1,125 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import {
+  Text,
+  IconButton,
+  NotificationMarker,
+  NotificationMarkerProps,
+} from '@itwin/itwinui-react';
+import SvgNotification from '@itwin/itwinui-icons-react/cjs/icons/Notification';
+
+export default {
+  component: NotificationMarker,
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+    type: {
+      control: 'radio',
+      options: ['primary', 'positive', 'warning', 'negative'],
+    },
+  },
+  title: 'Core/NotificationMarker',
+} as Meta<NotificationMarkerProps>;
+
+export const Basic: Story<NotificationMarkerProps> = (args) => {
+  return (
+    <IconButton styleType='borderless'>
+      <NotificationMarker {...args}>{args.children}</NotificationMarker>
+    </IconButton>
+  );
+};
+
+Basic.args = {
+  type: 'primary',
+  urgent: false,
+  children: <SvgNotification />,
+};
+
+Basic.argTypes = {
+  children: {
+    control: false,
+  },
+};
+
+export const CustomChild: Story<NotificationMarkerProps> = (args) => {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'flex-start',
+        gap: '20px',
+      }}
+    >
+      <NotificationMarker type='positive' {...args}>
+        Live
+      </NotificationMarker>
+      <NotificationMarker type='negative' {...args}>
+        Rec
+      </NotificationMarker>
+      <NotificationMarker type='primary' {...args}>
+        1 new message
+      </NotificationMarker>
+    </div>
+  );
+};
+
+CustomChild.args = {
+  urgent: true,
+};
+
+CustomChild.argTypes = {
+  type: {
+    control: false,
+  },
+};
+
+export const ProgrammaticDisplay: Story<
```

Comment:
I don't think this story is needed. Everything is "programmatic" if you think about it.

Reply:
I removed this `ProgrammaticDisplay` story. Do you think there should be documentation, a story, or something similar to show how to use the `enabled` (currently called `active`) prop? Because without mentioning it, people might do something like:

```jsx
{numNotifications > 0 ? <NotificationMarker>child</NotificationMarker> : child}
```

instead of:

```jsx
{<NotificationMarker enabled={numNotifications > 0}>child</NotificationMarker>}
```

Using the first way would lead to an extra dom `<div>` when numNotifications > 0. This might cause CSS styling problems (users' direct descendent styles with `>`) and also maybe longer untidy code

Reply:
You have a good example on the property itself. Not sure how else to disseminate that info with the current documentation setup we have.

Reply:
I don't think it's necessarily a big deal if users do a ternary to toggle the notification-marker, so i think it's fine if we don't even show an example tbh